test/katex-spec.js

```diff
@@ -3060,6 +3061,53 @@ describe("A parser that does not throw on unsupported commands", function() {
     });
 });
 
+describe("ParseError properties", function() {
+    it("should contain affected position and length information", function() {
+        try {
+            katex.renderToString("1 + \\fraq{}{}");
+
+            // Render is expected to throw, so this should not be called.
+            fail();
```

Comment:
The linter throws an error on the `fail` command I used in the tests, because this function appears to be defined in a different package called Jasmine. I could add that to the `.eslintrc` file (appears to work), but I'm not sure if that's desired. 

Alternatively I could replace fail with some other failing check like `expect(true).toBe(false)`. What do you prefer?